docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,572 @@
+* [How to target both .NET Core and Older .NET Framework Versions](#how-to-target-both-net-core-and-older-net-framework-versions)
+* [How to target a Portable Class Library (PCL)](#how to target a portable class-library-pcl)
+* [How to cross-compile for .NET Core and .NET Framework](#how-to-cross-compile-for-net-core-and-net-framework)
+* [How to test libraries on .NET Core](#how-to-test-libraries-on-net-core)
+* [How to create a NuGet Package with your Library](#how-to-create-a-nuget-package-with-your-library)
+
+## Prerequisites
+
+You must have .NET Core installed on your machine.  This guide will focus on command line tooling because they work cross-platform and are what Visual Studio uses internally.  You have two options:
+
+* [DNX](http://aspnet.readthedocs.org/en/latest/getting-started/index.html), which has shipped with Visual Studio 2015 Update 1
+
+* [.NET CLI](http://dotnet.github.io/getting-started/), the tooling which will eventually replace DNX
+
+Either toolchain can be used to build and publish libraries today.
+
+The sections of this document dealing with older frameworks or Portable Class Libraries (PCLs) require Windows and the .NET Framework installed.  The best way to do this is to [install Visual Studio](https://www.visualstudio.com/en-us/visual-studio-homepage-vs.aspx).  Alternatively, you can [install .NET Framework 4.6](https://www.microsoft.com/en-us/download/details.aspx?id=48130) as a standalone package.  This will contain all the dependencies required for targeting older frameworks and PCLs you may wish to support.
+
+## How to target .NET Core
+
+For the purposes of building a library, targeting ".NET Core" means targeting the .NET Standard Platform.  To see what that means exactly, see [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md).
+
+A distilled version is this:
+
+You need to pick a version of `netstandardX.X` (where `X.X` is a version number) to add to your `project.json` file.
+
+1. If you don't care about backwards compatibility, target `netstandard1.4`.
+2. If you care about compatibility with any of the following:
+
+    - .NET Framework 4.6.1, 4.6, 4.5.2, 4.5.1, or 4.5
+    - Windows Phone
+    - Windows Phone Silverlight
+    - Universal Windows Platform
+    - DNX Core
+    - Xamarin Platforms
+    - Mono 
+
+    Then [refer to this table](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md#mapping-the-net-platform-standard-to-platforms) and choose the version of the `dotnetXX` moniker you need.
+    
+    For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `netstandard1.4`:
+
+    ```javascript
+    {
+        "frameworks":{
+            "netstandard1.4":{}
+        }
+    }
+    ```
+    
+    If you wanted to be compatible with Windows Phone Silverlight 8, you would pick `netstandard1.1`:
```

Comment:
You should clarify that you also get other platforms with 1.1 and that you have a much more limited set of APIs this way.